hitch-2011/hitch-fe
# Feature/travis ci setup

## Is this a fix or a feature?
This is a feature

## What is the change?

Add cypress for testing with one sample test and install Travis CI.

## What does it fix?
N/a
## Where should the reviewer start?

In `package.json` file and `.yml` file

## How should this be tested?

Run `npm run cypress` the only sample test should pass.  This is test to make sure Travis Ci is working
## Relevant ScreenShots
